module/mock/hot_stuff_follower.go

```diff
@@ -1,4 +1,4 @@
-// Code generated by mockery v2.13.1. DO NOT EDIT.
```

Comment:
changes to this file weren't in the original

Reply:
https://github.com/onflow/flow-go/pull/3005/commits/1eb538b3a3fd7f9e77d8c06a65ff3e3bf5be9fda